### PR TITLE
Create Council Page

### LINKED ISSUE #19: Page: Council
Redevelop https://mathsoc.uwaterloo.ca/council/ for the new site, with CMS customization.

council

**Acceptance criteria:** 
* The page is rebuilt, containing all information from the version on the old site.
* The list of executives is built from JSON & image data stored on the server
* The list of executives can be updated from within the CMS

**Requires:** #5

### PR DESCRIPTION
**Pull request by ad-world**
This pr:
- Creates a page for MathSoc Countil
- Moves exec data, navbar data, footer data into shared folder. 
- Changes contact use page to use shared exec file

Closes #19

### CODE REVIEW

File: server/data/get-involved/council.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
...p;</span>which are open to all math students in Mathematics/Business programs, as well as all students in the Business Administration and Computer Science Double Degree program.</li><li><strong style=\" font-weight: 700\">One (1) Computing and Financial Management Representative,<span>&nbsp;</span></strong>which is open to all math students in the Computing and Financial Management program.</li><li><strong style=\" font-weight: 700\">Two (2) Software Engineering Representatives,</strong><span>&nbsp;</span>which are open to all math students in the Software Engineering program. The process to become a Software Engineering Representative is different from the other positions and is described in Bylaw 7.3.1.</li><li><strong style=\" font-weight: 700\">One (1) Teaching Representative,</strong><span>&nbsp;</span>which is open to all math students in the Mathematics/Teaching or Pure Mathematics/Teaching programs.</li><li><strong style=\" font-weight: 700\">Three (3) Mathematical Studies and Other Representatives,<span>&nbsp;</span></strong>which are open to all math students in Mathematical Studies programs and all math students not counted in one of the other constituencies.</li></ol>",
```
Comment (some-molasses) [inline server/data/get-involved/council.json:5]: Not really a requested change on this ticket, but we're going to need to find a way to handle editing these larger blocks of text.  I'm not sure what exactly the solution to that is though

Comment (ad-world) [reply server/data/get-involved/council.json:5]: i envision using a rich text editor and somehow attaching it to the json editor for all editable values

File: server/data/shared/navbar.json [hunk 2]
```diff
@@ -0,0 +1,148 @@
+[
+  {
+    "title": "Get involved",
```
[the remark this answers is not in the document]
Comment (ad-world) [reply server/data/shared/navbar.json:3]: Nope you're completely right, i'll move them back out

**Review by some-molasses — APPROVED** [review]
approved!

**Pull request merged**